packages/cloud2edge/Chart.yaml

```diff
@@ -11,8 +11,8 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 apiVersion: v1
-version: 0.4.2
-appVersion: 0.4.2
+version: 0.4.1
+appVersion: 0.4.1
```

Comment:
I do not think that the c2e package chart needs to be changed as part of this PR, or am I mistaken?

Reply:
I guess that was a rebase-error - the c2e chart had been updated to 0.4.2 in the meantime.

Reply:
It was indeed a rebase error I fixed it now